gtsfm/utils/metrics.py

```diff
@@ -61,14 +61,19 @@ def compute_correspondence_metrics(
     Returns:
         Boolean mask of which verified correspondences are classified as correct under Sampson error
             (using GT epipolar geometry).
+        Reprojection error for every verified correspondence against GT geometry.
     """
     if corr_idxs_i1i2.size == 0:
         return None, None
+    
+    if gt_wTi1 is None or gt_wTi2 is None:
+        return None, None
 
     # Compute ground truth correspondences.
     matched_keypoints_i1 = keypoints_i1.extract_indices(corr_idxs_i1i2[:, 0])
     matched_keypoints_i2 = keypoints_i2.extract_indices(corr_idxs_i1i2[:, 1])
-    if None not in [gt_scene_mesh, gt_wTi1, gt_wTi2]:
+    # check to see if a GT mesh is provided.
```

Comment:
nit: capitalize first letter of sentence

Reply:
thanks, updated.